packages/cdk/utils/src/replace-ranges-with-seconds/replace-with-numbers.ts

```diff
@@ -0,0 +1,64 @@
+import { Nullable } from '@angular-ru/cdk/typings';
+
+import { isNil } from '../is-nil';
+import { isNotNil } from '../is-not-nil';
+import { ReplaceNumber } from './types';
+
+const REGEXP_NUMBER: string = `[0-9]+([\.][0-9]+)?`;
+
+export function replaceWithNumbers(value: string, replaceMapList: ReplaceNumber[]): string {
```

Comment:
can we decompose this file to separate files?

Reply:
in general we can, but I think it is excessive. they are not exported, and all the logic is in one file before our eyes

Reply:
oh, not problem)